src/libs/actions/PersonalDetails.js

```diff
@@ -81,6 +82,24 @@ function extractFirstAndLastNameFromAvailableDetails({
     };
 }
 
+/**
+ * Convert country names to their respective ISO codes obtained from the backend
```

Comment:
Idk if "Obtained from the backend" is really true, since the ISO codes are hardcoded in CONST